common/searchattribute/mapper.go

```diff
@@ -219,7 +220,7 @@ func UnaliasFields(
 
 	// If no alias was mapped, return nil to save on clone operation on caller side.
 	if !mapped {
-		return nil, nil
+		return searchAttributes, nil
```

Comment:
If you want to change this, you also need to change the comment right here, and the one at the top of the function, to describe the new semantics